scripts/control_bar.js

```diff
@@ -359,6 +389,13 @@ function makeImageWidget(container, align = "C") {
         localStorage.setItem(imageKey, JSON.stringify({zoom: percent}));
     }
 
+    function reScroll () {
+        // keep same scroll proportion when contentWidth changes
+        let oldScroll = content.scrollLeft / contentWidth;
```

Comment:
I know it is an edge case, but do we need to worry about if contentWidth is 0? Say if someone drags too much on accident? I don't want to get in an exception case and the user is in some bad state.

Reply:
I'll look into this.

Reply:
Caught this case.